src/ps_collector/mesh.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import print_function
+
+import requests
+import json
+
+
+class Mesh:
+    def __init__(self, base_url = "https://psconfig.opensciencegrid.org/pub/config"):
+        self.base_url = base_url
+
+    def get_nodes(self):
+        """
+        Get all the nodes we should collect statistics
+
+        :returns list: List of nodes in the mesh
+        """
+        # Get the top level config, with list of URLs
+        meshes = self._download_toplevel()
+        nodes = set()
+        for mesh_url in meshes:
+            nodes.update(self._download_nodes(mesh_url))
+
+        return nodes
+    
+
+    def _download_nodes(self, mesh_url):
```

Comment:
I realized that I have always made a mistake of looking just at nodes on the mesh. This was right for most meshes. However it should be looking in the Json at `read_url` see for example [stashcache mesh](https://psconfig.opensciencegrid.org/pub/config/stashcache). The node `stashcache.t2.ucsd.edu` has its esmond on `perfsonar.nautilus.optiputer.net`. And this is where it should be getting its data. So basically should be looking at `read_url` and not at `members`